wIndex/QuickTools-Names.md

```diff
@@ -8,21 +8,53 @@ description: The Names Tools within Quick Tools deal with defined names.
 
 ## Overview
 
-The Names Tools within Quick Tools deal with defined names.
+The Names Tools within Quick Tools deal with defined names. Working with names is sometimes cumbersome within Excel. The Quick Tools makes the process easier and more convenient.
```

Comment:
it should be "the quick tools make this easier"
tense needs to match, and we don't need the "and more convenient"